### PR TITLE
stop click propagation added to the docs

### PR DESCRIPTION
**Pull request by 0000marcell**
issue #1069

### CODE REVIEW

File: tests/dummy/app/templates/public-pages/docs/the-list.hbs [hunk 1]
```diff
@@ -149,6 +149,16 @@
   This is because it deserves an entire chapter of its own.
 </p>
 
+<h3>Prevent click propagation inside the list</h3>
+
+<p>
+ The options inside the list are selected with mousedown, a good way to prevent the selection is intercepting onmouseup and calling stopPropagation on the element "e.stopPropagation()"
```
Comment (cibernox) [inline tests/dummy/app/templates/public-pages/docs/the-list.hbs]: I'd add a note saying something: "This is particularly useful if you want to add some kind of link or button inside the list of option and prevent the option that contains it from being selected when the user clicks on them"

File: tests/dummy/app/templates/snippets/the-list-10.hbs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{{#power-select
+  options=names
+  renderInPlace=true
+  selected=name345
+  onchange=(action (mut name345))
+  as |name|
+}}
+  {{name}}
+  <a onmouseup={{action "stopPropagation"}} 
+    onclick={{action "removeName" name}}>x</a>
```
Comment (cibernox) [inline tests/dummy/app/templates/snippets/the-list-10.hbs]: use `&times;` instead of `x`. It looks nicer

Comment (0000marcell) [reply tests/dummy/app/templates/snippets/the-list-10.hbs]: Noted Miguel, gonna make both changes pretty soon, do you know why the travis build is failing ? will that be a problem ?

**Comment by cibernox** [comment]
Thanks! I'll fix the test myself soon.

**Pull request merged**